ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/streaming/StreamedQueryResource.java

```diff
@@ -460,18 +488,17 @@ private EndpointResponse handleScalablePushQuery(
         new PushQueryConfigPlannerOptions(ksqlConfig, configOverrides);
 
     final ScalablePushQueryMetadata query = ksqlEngine
-        .executeScalablePushQuery(serviceContext, configured, pushRouting, routingOptions,
+        .executeScalablePushQuery(analysis, serviceContext, configured, pushRouting, routingOptions,
             plannerOptions, context);
 
-
     final QueryStreamWriter queryStreamWriter = new QueryStreamWriter(
         query,
         disconnectCheckInterval.toMillis(),
         OBJECT_MAPPER,
         connectionClosedFuture
     );
 
-    log.info("Streaming query '{}'", statement.getStatementText());
+    QueryLogger.info("Streaming query", statement.getStatementText());
```

Comment:
putting some thoughts down here: we probably want to have another PR that goes through and standardizes what we do for query logging across the transient query types (and also logs the messages meaningfully, e.g. there's no indication that this is a scalable push query from the query log). doesn't belong in this PR though, can you create a ticket to track this (tag `fix-it-week` and maybe also `p1`)

Reply:
Sure, I agree we should audit the codebase and make sure we always use the logger for queries. I thought we already had, so I was surprised to find this here.

Reply:
Filed https://github.com/confluentinc/ksql/issues/7979